### PR TITLE
Add blogs published on developers.redhat.com blog

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,8 @@
 # talks
 Talks, presentations and research papers published about Thoth
+
+# Blog posts
+
+ * [Use Kebechet machine learning to perform source code operations](https://developers.redhat.com/blog/2020/12/24/use-kebechet-machine-learning-to-perform-source-code-operations/)
+
+ * [AI software stack inspection with Thoth and TensorFlow](https://developers.redhat.com/blog/2020/09/30/ai-software-stack-inspection-with-thoth-and-tensorflow/)
```
Comment (pacospace) [inline README.md:8]: There is also: https://developers.redhat.com/blog/2019/10/28/microbenchmarks-for-ai-applications-using-red-hat-openshift-on-psi-in-project-thoth/?

Comment (fridex) [reply README.md:8]: Added :+1:

**Review by pacospace — COMMENTED** [review]
lgtm, thanks :100: @KPostOffice is the other blog published?

**Comment by fridex** [comment]
> lgtm, thanks  @KPostOffice is the other blog published?

I haven't find any. I think it's safe to add it later.

**Review by pacospace — APPROVED** [review]

**Pull request merged**